tests/e2e-pw/tests/mini-cart/mini-cart.block_theme.spec.ts

```diff
@@ -28,17 +33,77 @@ test.describe( `${ blockData.name } Block`, () => {
 		} );
 
 		test( 'should open the empty cart drawer', async ( { page } ) => {
-			const miniCartButton = await page.getByLabel(
-				'0 items in cart, total price of $0.00'
+			const miniCartButton = await getMiniCartButton( { page } );
+
+			await miniCartButton.click();
+
+			await expect(
+				page.locator( blockData.selectors.frontend.drawer ).first()
+			).toHaveText( 'Your cart is currently empty!' );
+		} );
+
+		test( 'should close the drawer when clicking on the close button', async ( {
+			page,
+		} ) => {
+			const miniCartButton = await getMiniCartButton( { page } );
+
+			await miniCartButton.click();
+
+			await expect(
+				page.locator( blockData.selectors.frontend.drawer ).first()
+			).toHaveText( 'Your cart is currently empty!' );
+
+			// Wait for the drawer to fully open.
+			await page.waitForSelector(
+				blockData.selectors.frontend.drawerCloseButton
+			);
+
+			const closeButton = await page.$(
+				blockData.selectors.frontend.drawerCloseButton
 			);
 
+			await closeButton?.click();
+
+			// Wait for the drawer to fully close.
+			await page.waitForSelector( blockData.selectors.frontend.drawer, {
+				state: 'detached',
+			} );
+
+			expect(
+				await page
+					.locator( blockData.selectors.frontend.drawer )
+					.count()
+			).toEqual( 0 );
+		} );
+
+		test( 'should close the drawer when clicking outside the drawer', async ( {
+			page,
+		} ) => {
+			const miniCartButton = await getMiniCartButton( { page } );
+
 			await miniCartButton.click();
 
 			await expect(
-				page
-					.locator( blockData.selectors.frontend.drawer as string )
-					.first()
+				page.locator( blockData.selectors.frontend.drawer ).first()
 			).toHaveText( 'Your cart is currently empty!' );
+
+			// Wait for the drawer to fully open.
+			await page.waitForSelector(
+				blockData.selectors.frontend.drawerCloseButton
+			);
+
+			await page.mouse.click( 50, 200 );
```

Comment:
Total nitpick. Out of curiosity.

Is there a reason for using:
`await page.mouse.click( 50, 200 );`
and later on:
`await page.click( 'text=Add to cart' );`

It probably would be better for consistency to just use `await page.click();` in both cases.

Reply:
I believe that is to simulate clicking outside of the drawer so it would close.

Reply:
Yeah, in general I think we should favor using `page.click()` everything. But as @roykho, in this specific case we want to simulate clicking a specific pixel outside the drawer, and AFAIK that can't be done with `page.click()`, that's why we use `page.mouse.click()`: https://playwright.dev/docs/api/class-mouse.

Reply:
@Aljullu Awesome, thank you for explaining!